themes/theme-gmd/extension-config.json

```diff
@@ -675,6 +675,15 @@
         "path": "/v1/shop/%(shopId)s/gui_sg_connect_reviews_info_config?parsed=true",
         "key": "value"
       }
+    },
+    "activatePriceGrouping": {
```

Comment:
i'd prefer `showPriceGrouping` as setting key, since `show` is the most common prefix in the config for settings that can be toggled.